### PR TITLE
Feature identical column

### PR DESCRIPTION
**Pull request by valydia**
Implemented new  identical rule to specify that all values of a same column must be identical between them, without specifying that value.

### CODE REVIEW

File: csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ */
+package uk.gov.nationalarchives.csv.validator.schema
+
+import org.specs2.mutable.Specification
+import uk.gov.nationalarchives.csv.validator.metadata.{Cell, Row}
+
+import scalaz.{Failure, Success}
+
+/**
+ * User: Jim Collins
```
Comment (adamretter) [inline csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala]: @valydia Are you having an identity crisis? or has Jim rejoined DRI ;-)

File: csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2013, The National Archives <digitalpreservation@nationalarchives.gov.uk>
```
Comment (adamretter) [inline csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala]: Date is wrong!

**Comment by adamretter** [comment]
@valydia Can you look at:

1. The two simple inline comments.
2. Why the specs in `uk.gov.nationalarchives.csv.validator.MetaDataValidatorAcceptanceSpec` are failing? (see Travis-CI)
3. I think you could probably squash the four commits into one as they are just adding the one feature...

**Comment by valydia** [comment]
Hi Adam,
I've taken into account all your comments except the one with the date in the copyright. We have a maven plugin that check all the license header are identical, so i left as it was for the time being, we may update all headers to 2013 - 2015, but we need to discuss this more.

**Comment by adamretter** [comment]
Okay no problem. Of course when you like, you can just run `mvn license:format` ;-)

File: csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala (hunk 1 above)
Comment (jim-collins) [inline csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/schema/IdenticalRuleSpec.scala]: No I am still at Paribas :-)

**Pull request closed without merging**